.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6.x', '2.5.x', '2.4.x', '2.3.x' ]
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: |
+        # As of 3 Jan 2019, Travis defaults to bundler 2.0 - https://docs.travis-ci.com/user/languages/ruby/#bundler-20
+        # Linguist still requires Bundler 1.x due to Licensed.
+        gem uninstall -v '>= 2' -ax bundler || true
+        gem install bundler -v '< 2'
+        # Fetch all commits/refs needed to run our tests.
+        git fetch origin master:master v2.0.0:v2.0.0 test/attributes:test/attributes test/master:test/master
+        # Replace SSH links to submodules by HTTPS links.
+        sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```

Comment:
The escaped backslashes can be eliminated by picking a different delimiter. E.g.:

~~~shell
sed -i 's|git@github.com:|https://github.com/|' .gitmodules
~~~